frontend/src/metabase/components/AddDatabaseHelpCard.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useMemo } from "react";
+import PropTypes from "prop-types";
+import { Flex } from "grid-styled";
+
+import { t, jt } from "ttag";
+
+import MetabaseSettings from "metabase/lib/settings";
+
+import ExternalLink from "metabase/components/ExternalLink";
+import Icon from "metabase/components/Icon";
+
+export const ENGINE_DOCS = {
+  bigquery: MetabaseSettings.docsUrl("administration-guide/databases/bigquery"),
+  mongo: MetabaseSettings.docsUrl("administration-guide/databases/mongodb"),
+  mysql: MetabaseSettings.docsUrl("administration-guide/databases/mysql"),
+  oracle: MetabaseSettings.docsUrl("administration-guide/databases/oracle"),
+  snowflake: MetabaseSettings.docsUrl(
+    "administration-guide/databases/snowflake",
+  ),
+  vertica: MetabaseSettings.docsUrl("administration-guide/databases/vertica"),
+};
+
+export const GENERAL_DB_DOC = MetabaseSettings.docsUrl(
+  "administration-guide/01-managing-databases",
+);
+
+export const CLOUD_HELP_URL = "https://www.metabase.com/help/cloud";
+
+const propTypes = {
+  engine: PropTypes.string.isRequired,
+  hasCircle: PropTypes.bool,
+  style: PropTypes.object,
+};
+
+const defaultProps = {
+  hasCircle: true,
+  style: {},
+};
+
+function AddDatabaseHelpCard({ engine, hasCircle, style, ...props }) {
+  const displayName = useMemo(() => {
+    const hasEngineDoc = !!ENGINE_DOCS[engine];
+    if (!hasEngineDoc) {
+      return "your database";
+    }
+    const engines = MetabaseSettings.get("engines");
+    return (engines[engine] || {})["driver-name"];
+  }, [engine]);
+
+  const docsLink = ENGINE_DOCS[engine] || GENERAL_DB_DOC;
+  const shouldDisplayHelpLink = MetabaseSettings.isHosted();
+
+  return (
+    <Flex
+      p={2}
+      style={{
+        backgroundColor: "#F9FBFB",
```

Comment:
NIT: Even though we have not discussed and agreed on the way we write styles, usage of [inline styles is discouraged](https://github.com/metabase/metabase/wiki/Frontend%3A-Component-Styling-Tree-Rings#inline-style-discouraged)

Reply:
Ouch, my bad. Thank you for point that out